api/v1beta1/cluster_types.go

```diff
@@ -206,6 +206,11 @@ type MachineDeploymentTopology struct {
 	// +optional
 	MinReadySeconds *int32 `json:"minReadySeconds,omitempty"`
 
+	// The deployment strategy to use to replace existing machines with
+	// new ones.
```

Comment:
As far as I can tell the godoc comments are both idential at the moment (?)

But it would be good to fix the godoc comment to something like
```go
// Strategy describes how to replace existing machines
// with new ones.
```

(comment should start with the field name not the struct name)

@killianmuldoon Can you maybe open a quick follow-up PR when you get to it? I can then approve it

Reply:
Just saw there are more incorrect comments in MD, e.g. Selector, Replicas, MinReadySeconds

Can you maybe open an issue, use the fields in MD as examples and then scope the issue to going over all our v1beta1 API types and fixing the "godoc comment should start with field name" findings?

Reply:
Also would be nice if we can take another look in the issue if there is a way to lint this (without using the godot linter which produces way too many false positives)

Reply:
Merci!